sdk/communication/azure-communication-jobrouter/CHANGELOG.md

```diff
@@ -130,7 +130,7 @@
 #### BestWorkerMode
 - Added setters to `ScoringRule` and `ScoringRuleOptions`
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2023-07-27)
```

Comment:
https://repo1.maven.org/maven2/com/azure/azure-communication-jobrouter/

1.0.0-beta.1 is the latest released version. 

We should remove ## 1.0.0-beta.2 sections. and update the release date for 1.0.0.

Reply:
Thank you. Updated it.